compiler/control/OMROptions.cpp

```diff
@@ -1953,7 +1953,7 @@ OMR::Options::realTimeGC()
    return self()->getOption(TR_RealTimeGC);
    }
 
-char *
+const char *
```

Comment:
It looks like there are references in [`CompileMethod.cpp`](https://github.com/dylanjtuttle/omr/blob/b2ae9f38c50e9a13a1c58d2a6d62106d18fe6972/compiler/control/CompileMethod.cpp#L193-L204) to `latePostProcessJIT` and `latePostProcessAOT` where the result is assigned to a `char *` variable.